presets/presets.go

```diff
@@ -71,8 +71,9 @@ func EVMReorg(config *environment.Config) *environment.Environment {
 				},
 			},
 		})).
-		AddHelm(chainlink.New(0, map[string]interface{}{
+		AddHelm(chainlink.NewVersioned(0, "0.0.11", map[string]interface{}{
```

Comment:
Any specific reason for this version?

Reply:
As far as I know this was the last helm chart version that accepted non toml configs.